apps/easypid/src/features/receive/slides/CredentialRetrievalSlide.tsx

```diff
@@ -25,19 +33,44 @@ import {
 } from 'react-native-reanimated'
 
 interface CredentialRetrievalSlideProps {
+  attributes: Record<string, unknown>
   display: CredentialDisplay
   isCompleted: boolean
+  onAccept: () => Promise<void>
   onGoToWallet: () => void
 }
 
-export const CredentialRetrievalSlide = ({ display, isCompleted, onGoToWallet }: CredentialRetrievalSlideProps) => {
-  const { completeProgressBar } = useWizard()
+export const CredentialRetrievalSlide = ({
```

Comment:
I know this file is a bit of a mess with all the states so I'm thinking about better abstractions for the animations.

But will prob be after the deadline